### PR TITLE
Add reading in different file type feature

### LINKED ISSUE #58: Parsing data from file of different formats
User Story:
- As an administrator, I want to upload student and allocation data in different formats (e.g., CSV, TXT), so that I can present the information in various settings or platforms.

Feature to implement:
- Parsing in data from files like .txt, .csv and .json

### PR DESCRIPTION
**Pull request by ehz0ah**
Close #58 

- User choose 1 out of the 2 options first presented to them when the program runs, which is to input manually or parse a file of type (.csv, .json or .txt)
- If file is read in successfully or user choose to input manually, the program goes back to normal functionality.

### CODE REVIEW

File: src/main/java/exceptions/SEPUnknownException.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    }
+
+    /**
+     * Returns a SEPEmptyException with a message indicating that the file type
+     * is unknown or unsupported.
+     *
+     * @return A SEPEmptyException with the appropriate message.
+     */
+    public static SEPEmptyException rejectUnknownFileType() {
+        return new SEPEmptyException("Unknown/Unsupported file type detected!");
```
Comment (Holy-An) [inline src/main/java/exceptions/SEPUnknownException.java:15]: You mean SEPUnkownException here instead of SEPEmptyException.

**Review by Holy-An — APPROVED** [review]
Overall, LGTM! Minor changes could be done

File: src/main/java/exceptions/SEPUnknownException.java (hunk 1 above)
Comment (ehz0ah) [reply src/main/java/exceptions/SEPUnknownException.java]: Nice catch!

**Review by Holy-An — APPROVED** [review]
LGTM

**Pull request merged**